CHANGELOG.md

```diff
@@ -15,6 +15,14 @@ The REST API versions in the table are just for your information as the values a
 
 - We ask developers to consider using the Headline component instead of the KPI component. The KPI component may be eventually marked as deprecated in one of the next major versions.
 
+## 6.3.1
+
+April 26, 2019
+
+### Changed
+
+- Use updated @gooddata/goodstrap library to fix error when installing dependencies on MS Windows machines ([commit](https://github.com/gooddata/gooddata-react-components/pull/929/commits/a6e4ab0acf91bb4318d8ffb2c394beae14adb125))
```

Comment:
Use the updated @gooddata/goodstrap library to fix the error that occurs when installing dependencies on MS Windows machines